project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ import sbt._
 object Dependencies {
   val scalaVersions = Seq("2.11.12", "2.12.10", "2.13.1")
 
-  val CapiModelsVersion = "15.9.8"
+  val CapiModelsVersion = "15.9.10"
```

Comment:
Based on https://github.com/guardian/content-api/pull/2556/files#diff-835bedc1586a84c60f1a1bf7f86a508d2494a56e65459b5abcbec31b15fc035d I guess we ought to bump this up to `15.9.12` too?

Reply:
I'll do that and see how we fare...